solutionbox/structured_data/datalab_solutions/structured_data/test/e2e_functions.py

```diff
@@ -88,35 +95,38 @@ def run_preprocess(output_dir, csv_filename, config_filename,
   print('Going to run command: %s' % ' '.join(cmd))
   subprocess.check_call(cmd, stderr=open(os.devnull, 'wb'))
 
-def run_training(output_dir, input_dir, config_filename, extra_args=[]):
-  """Runs Training via gcloud alpha ml local train.
+
+def run_training(output_dir, input_dir, schema_filename, transforms_filename,
+                 extra_args=[]):
+  """Runs Training via gcloud beta ml local train.
 
   Args:
     output_dir: the trainer's output folder
-    input_folder: should contain features_train*, features_eval*, and
+    input_dir: should contain features_train*, features_eval*, and
         mmetadata.json.
-    config_filename: path to the config file
+    schema_filename: path to the schema file
+    transforms_filename: path to the transforms file.
     extra_args: array of strings, passed to the trainer.
   """
   train_filename = os.path.join(input_dir, 'features_train*')
   eval_filename = os.path.join(input_dir, 'features_eval*')
   metadata_filename = os.path.join(input_dir, 'metadata.json')
-  cmd = ['gcloud alpha ml local train',
+
+  cmd = ['gcloud beta ml local train',
          '--module-name=trainer.task',
          '--package-path=trainer',
          '--',
          '--train_data_paths=%s' % train_filename,
          '--eval_data_paths=%s' % eval_filename,
          '--metadata_path=%s' % metadata_filename,
          '--output_path=%s' % output_dir,
-         '--transforms_config_file=%s' % config_filename,
+         '--schema_file=%s' % schema_filename,
+         '--transforms_file=%s' % transforms_filename,
          '--max_steps=2500'] + extra_args
   print('Current working directoyr: %s' % os.getcwd())
   print('Going to run command: %s' % ' '.join(cmd))
-  sp = subprocess.Popen(' '.join(cmd), shell=True, stderr=subprocess.PIPE) #open(os.devnull, 'wb'))
+  sp = subprocess.Popen(' '.join(cmd), shell=True, stderr=subprocess.PIPE) 
   _, err = sp.communicate()
   err = err.splitlines()
   print 'last line'
```

Comment:
yes. This is part of the tests, so the test does not print all the training output, just the last line that has the loss values.